src/SignalRServiceExtension/TriggerBindings/SignalRTriggerBinding.cs

```diff
@@ -0,0 +1,170 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
```

Comment:
extra line

Reply:
Keep the extra line from Zhenghui's suggestion. To avoid the auto reorder between system and other namespace